tutor/src/screens/assignment-builder/builder/tasking.js

```diff
@@ -167,11 +167,10 @@ class Tasking extends React.Component {
             </Col>
             <Col md={5} xs={4}>
               <TutorTimeInput
-                key={tasking.opensAtTime || 'opens_at_time'}
+                key={'opens-at-time'}
```

Comment:
This is totally my fault but now that I think about it, might be a good idea to add the `${type}` here (I think you can have multiple of these in the same page if you switch out of "All Sections")

Reply:
ah, yep.  good catch